src/app/routes/utils/constructPageFetchUrl/index.ts

```diff
@@ -0,0 +1,94 @@
+import Url from 'url-parse';
+import pipe from 'ramda/src/pipe';
```

Comment:
Is there any way we can get away from ramda functions here? (If not that's OK, I just had to ask 🙃 )

Reply:
We could, but this is one of the only Ramda functions I like as a concept from functional programming. Saves having to  to nest function calls or passing the result of functions to the next function invocation. Happy to remove if the general consensus is moving away from Ramda